cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -591,4 +643,35 @@ public String toString() {
       return CACHED_VALUES;
     }
   }
+
+  /**
+   * {@link _allUuidToUserTaskInfoMap} stores active and completed user tasks for history. Some historic user requests are
+   * less important for the user, e.g. {@link EndPoint.USER_TASKS} is not very interesting. To prevent un-interesting but
+   * frequently-called endpoints from flooding out interesting requests, we need to periodically cleanup requests. We can set
+   * per endpoint limit and utilize the {@link UserTaskScanner} to do the cleaning.
+   */
+  private class UserTaskCleanerInfo {
```

Comment:
@alexwang08 I believe this comment from my previous review was not addressed? To avoid redoing review for the same issue, can we either comment on a feedback to explain why it is not addressed or address the issue?

Reply:
Sorry missed this comment. I was hoping to discuss the semantic categorization further before deleting `UserTaskCleanerInfo` as it's needed for the current implementation.